bot/grab.py

```diff
@@ -82,7 +82,17 @@ def get_price(add_to_historical: bool = True) -> Dict:
     global historical_prices, hsp_head, session_struct
 
     initial_price = {}
-    prices = client.get_all_tickers()
+
+    # get all info on tickers from binance
+    # with retry on error reading
+    while True:
+        try:
+            prices = client.get_all_tickers()
+        except:
```

Comment:
Could we explicitly catch BinanceAPIException here?

Reply:
It's useless because there is only 1 function and I want to trap all error on this function... I don't know the quality of binance library